src/engines/kotor/gui/gui.cpp

```diff
@@ -93,11 +93,11 @@ Common::UString GUI::getName() const {
 }
 
 void GUI::mouseDown() {
-	CursorMan.setState("down");
+	CursorMan.set(CursorMan.getCurrentGroup(), "down");
```

Comment:
Huh...

`setState()` is supposed to do that, setting the cursor to another state in the current group.
Likewise, `setGroup()` is supposed to set a new group, with the current state.

The comments in cursorman.h even say that. However, the code does something else.
I wrote both, so I wonder what happened. :P

In either case, I think the code in cursorman.cpp should be changed to actually do what it claims to do.